src/main/java/org/myrobotlab/service/InMoov2.java

```diff
@@ -714,6 +802,7 @@ public boolean accept(File dir, String name) {
         for (File file : files) {
           Python p = (Python) Runtime.start("python", "Python");
           if (p != null) {
+            // FIXME error("x") when an error occurs
             p.execFile(file.getAbsolutePath());
```

Comment:
I think it would be best if we used invoke() here, should still maintain the order of files but it leverages the multithreaded pub/sub system to prevent InMoov's inbox thread from blocking for awhile